libresin/tests/files_helper.hpp

```diff
@@ -24,8 +26,11 @@ ::testing::AssertionResult IsFileContentEqual(const char* expected_expr, const c
   }
 
   std::string file_content((std::istreambuf_iterator<char>(file)), std::istreambuf_iterator<char>());
+  file_content = std::regex_replace(file_content, std::regex("\r\n"), "\n");
 
   if (file_content != content) {
+    std::println("|{}|", file_content);
```

Comment:
Use `::testing::AssertionFailure() << `

Reply:
this was purely for debug and idk if shouldn't be deleted entirely?

Reply:
if so then delete it